src/_P049_MHZ19.ino

```diff
@@ -445,13 +455,15 @@ boolean Plugin_049(byte function, struct EventStruct *event, String& string)
               addLog(LOG_LEVEL_INFO, log);
               break;
 
+//#ifdef ENABLE_DETECTION_RANGE_COMMANDS
```

Comment:
This #ifdef also in comments?

Reply:
Yes, don't you like it?
My intention was to indicate it's somehow disabled functionality, but still valid. In case of the Chinese firmware sends this response for some reason, it is still evaluated. Also, if an "universal" command function is added (which allows sending arbitrary commands), it is useful.

Reply:
I get why you do it and also like this approach.
But the other #ifdef statements are just that and the last one is placed in comment, effectively removing the #ifdef functionality of those lines and thus the lines inbetween will always be compiled regardless of the defined state.

Reply:
Good, but sorry, I think we have a misunderstanding: I intended that the code in between the commented #ifdefs will be included in the binary, independent whether the define is active or not, for the reason explained in my previous comment. I could remove it, but my intention was to keep it, but indicating, that it might be redundant.

Reply:
--> The commented #ifdef is introduced for documentation purpose. I want to indicate that it belongs so disabled functionality, but keep its functionality in, independent of the state of the #define.